hnn_core/tests/test_parallel_backends.py

```diff
@@ -32,6 +33,35 @@ def _terminate_mpibackend(event, backend):
         sleep(0.01)
 
 
+def test_gid_assignment():
+    """Test that gids are assigned without overlap across ranks"""
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+    params.update({'N_pyr_x': 3,
+                   'N_pyr_y': 3,
+                   'tstop': 40,
+                   't_evprox_1': 5,
+                   't_evdist_1': 10,
+                   't_evprox_2': 20,
+                   'N_trials': 2})
+    net = jones_2009_model(params, add_drives_from_params=True)
```

Comment:
```suggestion
    net = jones_2009_model(params, add_drives_from_params=False)
```

you might want to use `_gid_list` for the test. Add a drive explicitly and check if it contains all the `gids` in one rank if it's non-cell-specific etc.

Reply:
Done. See https://github.com/jonescompneurolab/hnn-core/pull/399/commits/cce3bd9f32706029f80a6e8bc39dc15ba29b5689.